### DISCUSSION TITLE
Please make youtube video on CustomErro page

### DISCUSSION THREAD

**Discussion by jitendravn** [nylo-core/nylo#134] [General] [answered]
![Screenshot 2024-04-29 at 10 51 41 AM](https://github.com/nylo-core/nylo/assets/93908273/36d85818-dab4-46b3-9ab7-63913ae0044d)

**Comment by agordn52** [comment, marked as answer]
Hi @jitendravn,

I think we'll have an option for custom error handling soon. It would be a cool idea.

Maybe from the router you'll be able to set this, we'll see.
